assignment.js

```diff
@@ -31,7 +31,13 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 1:')
 
 // Add your code below this line
+const baseNumber = 2
 
+//Used for loop to go through 12 levels
+for(i = 0; i <= 12; i++) {
+  //Each level multiplies by the base number and prints results to console
+  console.log(baseNumber * i)
+}
 // Add your code above this line
 
```

Comment:
@cswebdesign Good Job here using the for loop to loop through each value and multiply each value by two to arrive at your solution. Take a look at this approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L31)